examples/integrations/hf_trainer_api/adaptive.yaml

```diff
@@ -0,0 +1,49 @@
+name: vision transformer adaptive
```

Comment:
do we actually find a configuration that outperforms the default hyperparameter setting with asha?

Reply:
Not really. The reason is that the key hyperparameter here is the learning rate, and it is already set to the optimal 2e-5 value in the HF example. To make our example more convincing, I changed it to 1e-5 so that adaptive may claim the win.